docs/source/callbacks.rst

```diff
@@ -98,6 +98,7 @@ Lightning has a few built-in callbacks.
     EarlyStopping
```

Comment:
@tchaton did you forget to include `BackboneLambdaFinetuningCallback` here, or is it on purpose?

Reply:
My IDE warn to me because `lambda` is python built-in identifier.
So, below code raise error.
```
from pytorch_lightning.callbacks.lambda import LambdaCallback
```
Therefore I cannot change file name from `lambda_cb.py` to `lambda.py` 😢

Reply:
hmm... I use vim, so can't say what to do here. cc @awaelchli @SkafteNicki

Reply:
yes, it seems this built in name cannot be used in an import statement. It will be a syntax error.

```bash
  File "C:\Users\aeduw\Documents\Repositories\pytorch-lightning\pytorch_lightning\callbacks\__init__.py", line 21
    from pytorch_lightning.callbacks.lambda import LambdaCallback
                                          ^
SyntaxError: invalid syntax

```
No way around it, we cannot name it lambda.py :( 
Could consider name it `lambda_function.py`
or just misspelling it `lamda.py` 🤣

Reply:
Interestingly, we can have a file print.py and import it without any problem :)

Reply:
I think 'lambda_function.py' would be betther than 'lamda.my' 😀 
I will update it

Reply:
I updated it 👍